CODEOWNERS

```diff
@@ -402,6 +402,7 @@ Makefile* @cilium/build
 /Documentation/network/kubernetes/kubeproxy-free.rst @cilium/sig-lb @cilium/docs-structure
 /Documentation/network/kubernetes/local-redirect-policy.rst @cilium/sig-lb @cilium/docs-structure
 /Documentation/network/lb-ipam.rst @cilium/sig-lb @cilium/docs-structure
+/Documentation/network/concepts/masquerading.rst @cilium/sig-datapath @cilium/docs-structure
```

Comment:
nit: I think alphabetically this should be closer to `concepts/ipam/eni*`, but I don't think it matters much.

Reply:
Fixed.